test/systems/plonk/placeholder/placeholder.cpp

```diff
@@ -343,115 +341,117 @@ ACTOR_THREAD_TEST_CASE(placeholder_permutation_argument_test) {
-                                     placeholder_test_params_lookups::public_input_columns + i;
-
-        for (int rotation : preprocessed_public_data.common_data.columns_rotations[i_global_index]) {
-            auto key = std::make_tuple(i, rotation, plonk_variable<FieldType>::column_type::constant);
-
-            columns_at_y[key] = polynomial_table.constant(i).evaluate(y * circuit.omega.pow(rotation));
-        }
-    }
-    for (std::size_t i = 0; i < placeholder_test_params_lookups::selector_columns; i++) {
-
-        std::size_t i_global_index = placeholder_test_params_lookups::witness_columns +
-                                     placeholder_test_params_lookups::constant_columns +
-                                     placeholder_test_params_lookups::public_input_columns + i;
-
-        for (int rotation : preprocessed_public_data.common_data.columns_rotations[i_global_index]) {
-            auto key = std::make_tuple(i, rotation, plonk_variable<FieldType>::column_type::selector);
-
-            columns_at_y[key] = polynomial_table.selector(i).evaluate(y * circuit.omega.pow(rotation));
-        }
-    }
-
-    typename FieldType::value_type input_at_y = prover_res.input_polynomial.evaluate(y);
-    typename FieldType::value_type input_at_y_shifted =
-        prover_res.input_polynomial.evaluate(circuit.omega.inversed() * y);
-
-    typename FieldType::value_type value_at_y = prover_res.value_polynomial.evaluate(y);
-
-    typename FieldType::value_type v_l_at_y = prover_res.V_L_polynomial.evaluate(y);
-    typename FieldType::value_type v_l_at_y_shifted = prover_res.V_L_polynomial.evaluate(circuit.omega * y);
-
-    std::array<typename FieldType::value_type, 5> verifier_res =
-        placeholder_lookup_argument<FieldType, lpc_type, circuit_3_params>::verify_eval(
-            preprocessed_public_data, constraint_system.lookup_gates(), y, columns_at_y, input_at_y, input_at_y_shifted,
-            value_at_y, v_l_at_y, v_l_at_y_shifted, prover_res.input_precommitment.root(),
-            prover_res.value_precommitment.root(), prover_res.V_L_precommitment.root(), verifier_transcript);
-
-    typename FieldType::value_type verifier_next_challenge = verifier_transcript.template challenge<FieldType>();
-    typename FieldType::value_type prover_next_challenge = prover_transcript.template challenge<FieldType>();
-    BOOST_CHECK(verifier_next_challenge == prover_next_challenge);
-
-    for (int i = 0; i < argument_size; i++) {
-        BOOST_CHECK(prover_res.F[i].evaluate(y) == verifier_res[i]);
-        for (std::size_t j = 0; j < desc.rows_amount; j++) {
-            BOOST_CHECK(prover_res.F[i].evaluate(preprocessed_public_data.common_data.basic_domain->get_domain_element(
-                            j)) == FieldType::value_type::zero());
-        }
-    }
-}
+// This test is disabled in singlethreaded version due to a bug in zk/snark/systems/plonk/placeholder/lookup_argument.hpp line 160-173.
```

Comment:
Let's use ?
```
#ifdef 0
...
#endif
```

It's look little bit beatufily than many line of commendet code.

Reply:
Apparently you can not ifdef 0, so I added #ifdef ENABLE_PLACEHOLDER_LOOKUP_TESTS

Reply:
Oh, yeah, u right. 
```
#if 0
```

But anyway if we have something like #ifdef ENABLE_PLACEHOLDER_LOOKUP_TESTS it is much better!